rp_interceptors/tests.py

```diff
@@ -23,11 +23,13 @@ def setUp(self):
 
     def test_hmac_missing(self):
         """
-        If the HMAC secret is configured, and there's no HMAC header, or it's invalid,
-        then we should return a 403
+        If the HMAC secret is configured, and there's no HMAC header, or it's
+        invalid, then we should return a 403
         """
         interceptor = Interceptor.objects.create(
-            org=self.org, hmac_secret="test-secret", channel_uuid="1234343212"
+            org=self.org,
+            hmac_secret="test-secret",
+            channel_uuid="1234343212",  # noqa: S106 - Hardcoded passwords OK for tests
```

Comment:
Can't we move this ignore to the `pyproject.toml` file?

Reply:
The purpose of that particular check is to find places where we accidentally hardcoded creds. Ignoring these in-place (with an explanation for each) means we'll get warnings for new ones which may be problems -- if we ignore it globally in the config, we don't.

It would be nice if we could set a global ignore for `channel_uuid`, because that's never going to be a password, but the lints aren't configurable at that level.

Reply:
I see, makes sense, someone might accidentally put a real password in tests then the global rule ignores it.

Reply:
After discussion with @jerith, decided to keep the inline ignores for this